ros_speech_recognition/src/ros_speech_recognition/recognize_vosk.py

```diff
@@ -0,0 +1,37 @@
+# file to override recognize_vosk
+# we need this to use vosk model anywhere
+
+from speech_recognition import *
```

Comment:
It may be better to change, though it's a minor point.
```suggestion
from speech_recognition import AudioData
```